src/combineReducers.js

```diff
@@ -3,27 +3,27 @@ import warning from './utils/warning'
 import isPlainObject from './utils/isPlainObject'
 
 function getUndefinedStateErrorMessage(key, action) {
-  const actionType = action && action.type
+  const actionType= action && action.type
```

Comment:
format: style mismatch:
```suggestion
  const actionType = action && action.type
```

`ea11337e` ␣ should be inserted at column 19.